x-pack/plugins/apm/server/lib/search_strategies/correlations/queries/get_request_base.test.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getRequestBase } from './get_request_base';
+
+describe('correlations', () => {
+  describe('getRequestBase', () => {
+    it('returns the request base parameters', () => {
+      const requestBase = getRequestBase({
+        index: 'apm-*',
+        includeFrozen: true,
+      });
+      expect(requestBase).toEqual({
+        index: 'apm-*',
+        ignore_throttled: false,
+        ignore_unavailable: true,
```

Comment:
Just wanted to clarify, if `includeFrozen` is omitted or undefined here should the result still return `ignore_throttled: true`?

Reply:
I updated the code to be more in line with the rest of APM to default to `includeFrozen = false / ignore_throttled: true`, also added a test in 933a5d6eb4a.